release-lockfile.spec.md

```diff
@@ -148,6 +148,34 @@ information as to help differentiate the two deployed instances in a meaningful
 way.
 
 
+### Package Name
+
+A string matching the regular expression `[a-zA-Z][-_a-zA-Z0-9]{0,213}`
```

Comment:
Considering moving this up to allow 256 length as well.  214 was picked arbitrarily from the upper limit that NPM enforces.

Reply:
Stick with 256, it's more uniform that way.